bundles/org.openhab.binding.squeezebox/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -298,4 +300,14 @@
 		<description>Number of playlist tracks</description>
 		<state readOnly="true" pattern="%d"></state>
 	</channel-type>
+	<channel-type id="like" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Like Song</label>
+		<description>Likes the current song (if the service supports it)</description>
+	</channel-type>
+	<channel-type id="unlike" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Unlike Song</label>
+		<description>Unlikes the current song (if the service supports it)</description>
+	</channel-type>
```

Comment:
After reading the above discussion I agree with representing like/unlike with a single command channel. I understand that there might be UI issues regarding dynamic updating but I think it would be the correct direction to go here. For now you can just always provide the "like/unlike" commands and just ignore invalid requests.

Reply:
Ok, I'll change it to one channel. That's not an issue. It could go either way.

But, I'm not going to use command options because the UI issues are too severe, and there's no support for it in HABpanel.